app/static/js/events/event-base.js

```diff
@@ -0,0 +1,16 @@
+var deleteButton = $('#delete-event-button');
+var deleteCheck = $('#delete-check');
+var eventName = deleteCheck.attr('data-event-name');
+
+deleteButton.addClass('disabled');
+deleteCheck.keyup(function(){
+     if(deleteCheck.val() === eventName)
```

Comment:
Please follow a uniform code style. The bracket should start in the same line as the if. And make sure the ending bracket has the same indentation as if.